charts/nri-kube-events/templates/_helpers_compatibility.tpl

```diff
@@ -0,0 +1,8 @@
+{{/*
+Returns a dictionary with legacy runAsUser config
+*/}}
+{{- define "newrelic.compatibility.securityContext" -}}
+{{- if  .Values.runAsUser -}}
```

Comment:
could you add a unit test to check for the compatibility layer ?

Reply:
Added them here f39d852f58ab2cb7a03122d1e02722feec1ca659